[Please review me] Operation: Funky Nostaligia 2. Changes charliestation medkit icon, and changes heirloom item for MDs to an old medkit.

trollbreeder opened the pull request:
Funky nostaligia 1 failed because i'm a clown

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

1. Changes charliestation ancient medkits (those with bruise packs and ointment) to use the old medkit icon.
2. Adds a new family heirloom to MDs, PMs and CMOs: an old, empty medkit. It's empty because of assistants who kept breaking into medbay storage to heal themselves. This also removes the bodybag and stethoscope heirloom from MDs, CMOs and PMs in place of this medkit.

What's different this time? Well, i made this from a (hopefully recent enough) master and there's no horrific merge conflict frankenstein abominations this time. **Oh, and MDs, CMOs and PMs' heirlooms have all been removed in place of this ancient medkit.**

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

1. When i think of charliestation, i think of so old that the designs are from 2553. These ancient medkits use the normal medkit icon and that was a lost opportunity to me, so that is now rectified.
2. I would've expected more from the heirloom of MDs, who had to cope with a lot of changes, but apparantly not, just a body bag and a stethoscope. That's now rectified and MDs can now bring a piece of history along with them.

## Changelog
:cl:
add: Nanotrasen has asked to return all first-aid kits produced before 2559 as they've been recalled due to potential risk of lead acetate poisoning.
add: Medical doctors have been seen emotionally attached to these outdated first-aid kits.
del: Nanotrasen reports fewer cases of necrophillia, as medical doctors no longer feel any emotions towards body bags.
del: Nanotrasen also reports that medical doctors are no longer attached to 21st century heart listening devices.
imageadd: Ancient medkits are now properly ancient in design.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

trollbreeder committed:
Operation: Funky Nostaligia 2

the last one failed because i'm a clown

ExcessiveUseOfCobblestone commented on code/game/objects/items/storage/firstaid.dm:
```diff
@@ -114,7 +114,7 @@
 
 /obj/item/storage/firstaid/ancient
-	icon_state = "firstaid"
+	icon_state = "oldifrstaid" // Alan please correct this typo because i have no idea how merges work
```
time to learn sweetie!

trollbreeder replied:
hooray i've learned

Fikou commented:
nostaligia

trollbreeder committed:
Fix typo in icon state

Please dont kill me if this causes merge conflix because this happened the last time i tried doing this

trollbreeder committed:
Merge branch 'master' into funkynostaligia2

trollbreeder commented:
hooray, i've figured out how to update my local

trollbreeder commented:
ahh bugger more conflicts

trollbreeder commented:
okay nope

trollbreeder commented:
DNM while i revert

trollbreeder commented:
T-that should f-fix it right?! H-hopefully????

trollbreeder committed:
Merge branch 'master' into funkynostaligia2

Attempt 2 at fixing conflicts, hopefully i dont fuck shit up again

trollbreeder committed:
GET THE HELL OUT, MERGE CONFLICTS
